### DISCUSSION TITLE
Ventura-Monterey-BigSur-OC-0.8.4v2 Release- Infinix

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#10] [Discuss about the release]
Installation Instructions: [Here](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide#bios)

- With OpeCore-0.8.4
- Fixed Heating Issue. Now it's stable like windows. (For my usage, I see no heating at all even while charging).
- Increased Battery Backup. You may get more 30- 40 minutes of battery backup, _varies by usage._
- All 4 speakers are working now. Tested in Ventura.
- Fixed **FILE VAULT** feature. This works with any macOS Version.[proof](https://t.me/infinix_inbook_Updates/48)
- Increased Stability.
- Fixed **Volume Has Mishmash**. Tested in Only Ventura beta 6. _(this issue is common and may come again.)_
- Re-worked _ACPI_  session.
- **Donation will be helpful for future Updates:** [Here](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide#any-one-can-donate-me-or-buy-me-a-coffee-)
- Updated all the kexts.
- Removed _verbose-mode_(which shows the text while booting). Now it shows the apple logo in place of it.
- Updated required _Trackpad_ kexts too.
- Removed Unnecessary stuff in the EFI.
- Removed _CleanNVRAM.efi_. _ResetNVRAM_ is enough all the time.
- Reworked with _Power Management_.

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/8.4v2-infinix-hackintosh'>Ventura-Monterey-BigSur-OC-0.8.4v2 Release- Infinix</a>.</em>

**Comment by extasea** [comment]
Hi @devboloji thanks for updating the EFI for us
I just updated my system to Monterey 12.5.1 and use this Monterey EFI.
The system stuck at startup Apple logo (loading startup)
Then, I use the previous 0.8.4 Monterey EFI and the system boot normally.

Did I missed something?
Thanks,

**Reply by devboloji** [reply]
have you tried [Re-released](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/8.4v2-infinix-hackintosh) version of it?

**Reply by extasea** [reply]
I did used the re-release version, which is the latest one now available

**Reply by devboloji** [reply]
@extasea you better report in this [Telegram-Group](https://t.me/infinix_inbook_discussion)
